### ISSUE TITLE
Tracker Hide during Combat Uses Border Options

### ISSUE THREAD

**Issue by Starlynk1** [Questie/Questie#4957]
## Description

Currently if you have Hide during Combat turned on, the tracker minimizes but loses the border. Would like to see the border stay active when it minimizes. Border should shrink around the Header text, not disappear completely.

**Comment by Dyaxler** [comment]
I'm not against adding more options to make the Tracker as customizable as possible but right now, we have WAY too many options to choose from in the Questie Config --> Tracker tab. We're in the midst of discussing a redesign/reorganization of how those options are laid out and I have a few ideas on how to trim those options even further and move them to a new area. I've been toying with the idea of putting them into a "Right Click" menu on the Tracker itself, so you don't have to open Questie Config.

Stay tuned!

**Comment by Starlynk1** [comment]
Understand not wanting to add more options if you don't need to. Just seems odd that the border and background go away while in combat if you have it set to minimize in combat.

As for the menu idea, I think that would be great. Maybe you could reuse the minimap left click menu code for the tracker menu. You could use the tracker ? logo to left click and get menu. Just a thought, look forward to seeing how it ends up. thanks!

**Comment by Dyaxler** [comment]
> Understand not wanting to add more options if you don't need to. Just seems odd that the border and background go away while in combat if you have it set to minimize in combat.

All it does is minimize the Tracker for you when you enter combat. It's the same function if you were to manually minimize the Tracker. I.E. the Tracker looks the same in both use cases. Try minimizing the Tracker when you're not in combat. Click the Active Quests Header to min/max the Tracker and you'll see that they both look the same. Plus, not everyone uses the border and background so while minimized, we want the Tracker to look the same and be very minimalistic in appearance.

> As for the menu idea, I think that would be great. Maybe you could reuse the minimap left click menu code for the tracker menu. You could use the tracker ? logo to left click and get menu. Just a thought, look forward to seeing how it ends up. thanks!

Hmmmm... now there is an idea. I implemented the "?" mouseover tooltip before there was even a Minimap icon. The menu for that was added later. That idea didn't even occur to me. I've been looking for ways to move that tooltip somewhere else in case a player decides to hide the Active Quests Header. Putting the same menu on a "Right Click" inside the Tracker frame while the mouse pointer is NOT over a Quest... that might work. I could move all the Tooltip info over to the Minimap icon completely.

Thanks for the idea!
